packages/eslint-plugin/docs/rules/explicit-module-boundary-types.mdx

```diff
@@ -260,6 +260,37 @@ export const foo: FooType = bar => {};
 </TabItem>
 </Tabs>
 
+### `allowOverloadFunctions`
+
+{/* insert option description */}
+
+Examples of code for this rule with `{ allowOverloadFunctions: false }`:
```

Comment:
[Docs] The code blocks have the same content for _Incorrect_ and _Correct_, which as a reader I find confusing. But I can't think of anything to put in the _Correct_ one. Maybe it'd be best to just have one code block, not two tabbed ones?

Reply:
Yes!

I agree, I've updated the PR to only show one code block for correct code, as the option expands what the rule allows.